hangman.js

```diff
@@ -1 +1,76 @@
-console.log("welcome to hangman");
\ No newline at end of file
+//Welcome Message
+console.log("welcome to hangman");
+// Prompts user to get secret word --> UNCOMMENT WHEN DONE <-- secretWord = prompt("Please select a secret word!")
+
+  //Hardcoded secretWord, delete when done.
+  var secretWord = 'toast'
+  console.log('secretWord = ' + secretWord)
+
+secretWordHidden = []
+letterGuesses = []
+
+secretWordArray = secretWord.split('');
+
+
+// Function to show user the amount of letters in the secret word
+function spaces(){
+  for (i = 0; i < secretWord.length; i++) {
+    secretWordHidden.push('_');
+  }
+} spaces()
+
+//Loop to start the letter guessing protocol
+for (i = 0; (i < (secretWord.length)); i++) {
+  //Executes prompt
+  letterGuesses.push(prompt('Word:  ' + secretWordHidden.join(' ')+ '\n' + 'Please guess a letter.  Guesses left: ' + ((secretWord.length + 1)- i)))
+  console.log(i + ' time running ; letterGuessed = ' + letterGuesses[i] + 'letterGuesses = ' + letterGuesses)
+  //2nd loop to evaluate if the guess was correct
+  if (secretWord.includes(letterGuesses[i]) === true) {
+      //If the letter guessed is in the secretWord, the 1st IF statement evulates here to the 2nd loop
+      for (j = 0; j < secretWord.length; j++) {
```

Comment:
I think this nested for loop is throwing things off a little.. it looks like it's supposed to loop over the same length as the first loop? let me know if you want to talk throguh this live!